app/controllers/games_controller.rb

```diff
@@ -35,6 +36,11 @@ def destroy
     redirect_to games_path, notice: 'Your game has been cancel'
   end
 
+  def move_to!
```

Comment:
I think this is included by mistake. 

Duplicate code found in pieces controller. The piece_params you try to pass into `@piece.update_attributes` isn't accessible here. In a private method in the other controller...